assets/tests/hooks/useDetour.test.ts

```diff
@@ -293,65 +127,10 @@ describe("useDetour", () => {
     act(() => result.current.clear?.())
 
     await waitFor(() => {
-      expect(result.current.waypoints).toHaveLength(0)
-      expect(result.current.directions).toBeUndefined()
       expect(result.current.detourShape).toHaveLength(0)
     })
   })
 
-  test("when `startPoint` is null, `canUndo` is `false`", async () => {
-    const { result } = renderHook(useDetourWithFakeRoutePattern)
-
-    await waitFor(() => {
-      expect(result.current.startPoint).toBeNull()
-      expect(result.current.canUndo).toBe(false)
-    })
-  })
```

Comment:
Agreed - thanks for catching that!

Reply:
[Feedback: Add missing coverage that 'Undo' and 'Clear' get disabled when the detour is fully cleared out](https://github.com/mbta/skate/pull/2580/commits/e8b491b850dfb2ad8d0f3f7b5aa8ddf91f7e90bc)

☝️ adds both the use case you mentioned, and also a similar one when the detour is cleared because of the `Clear` button.